src/stan/lang/generator/generate_validate_nonnegative.hpp

```diff
@@ -18,12 +18,8 @@ namespace lang {
  * This check should precede the variable declaration in order to
  * avoid bad alloc runtime error.
  * Called by
- * <br />generate_validate_context_size - data variables
- * <br />generate_initialization - transformed data declarations
- * <br />generate_var_resiszing - initializes transformed data variables
- * <br />generate_local_var_decl_inits - local variables, transformed parameters
- *                              write array, generated quantities
- * <br/> generate_set_param_ranges - parameter variables
+ * <br />generate_validate_var_dims
+ * <br/> generate_set_param_ranges
  *
```

Comment:
This list was out of date. I updated it, but I think we should just remove it.

Reply:
Thanks. This code is no longer being maintained as the stan compiler written in c++ (src/stan/lang folder) is currently only being used to debug stanc3, but I guess it doesnt hurt to clean this up.

Reply:
Thanks for explaining that. On the plus side, documentation can't go out of sync with code that isn't changing anymore :).